docs/tsg_iscsi.md

```diff
@@ -7,9 +7,46 @@ sidebar_label: iSCSI
 ------
 
 ## Issue:
+**The iscsi login is working from host/node but When done from within the container-it fails.**
 
+## Troubleshooting the issue and Workaround:
+**If below error is seen when trying to login from kubelet:**
+```
+root@worker3:/# iscsiadm -m discovery -t st -p 10.43.21.74:3260
+10.43.21.74:3260,1 iqn.2016-09.com.openebs.jiva:pvc-641131af-75e2-11e8-990b-9600000c0999
+root@worker3:/# iscsiadm -m node -l -T iqn.2016-09.com.openebs.jiva:pvc-641131af-75e2-11e8-990b-9600000c0999 -p 10.43.21.74:3260
+Logging in to [iface: default, target: iqn.2016-09.com.openebs.jiva:pvc-641131af-75e2-11e8-990b-9600000c0999, portal: 10.43.21.74,3260] (multiple)
+iscsiadm: Could not login to [iface: default, target: iqn.2016-09.com.openebs.jiva:pvc-641131af-75e2-11e8-990b-9600000c0999, portal: 10.43.21.74,3260].
+iscsiadm: initiator reported error (12 - iSCSI driver not found. Please make sure it is loaded, and retry the operation)
+iscsiadm: Could not log into all portals
+```
+**Then the issue is with difference in version between the kubelet and node. The following command can be used to determine this:**
+```
+root@worker1 ~ # iscsiadm -V
+iscsiadm version 2.0-873
+root@worker1 ~ # sudo docker exec kubelet iscsiadm -V
+iscsiadm version 2.0-874
+```
+**Remove the iscsi from the host/node by using the following command:**
+```
+service iscsid stop
+sudo apt remove open-iscsi
+```
+**The above command will only remove iscsi from node. But the iscsi inside the kubelet will be running.Verify the same from the following command**
```

Comment:
The above command only removes iSCSI from the node, but iSCSI inside the kubelet will be running. Verify using the following command.